src/core/CompUnit/Repository/Installation.pm

```diff
@@ -367,11 +371,10 @@ sub MAIN(:$name is copy, :$auth, :$ver, *@, *%) {
                         (id => $_.basename, ver => Version.new( $ver || 0 ), auth => $auth, api => $api).hash
                     });
             for @dists.grep({$_<auth> ~~ $auth and $_<ver> ~~ $ver}) -> $dist is copy {
-                $dist = self!read-dist($dist<id>) if $version >= 1;
+                $dist = self!read-dist($dist<id>) if $repo-version >= 2;
                 with $dist<files>{$file} {
-                    my $candi   = %$dist;
-                    $candi<files>{$file} = $prefix.abspath ~ '/resources/' ~ $candi<files>{$file}
-                        unless $candi<files>{$file} ~~ /^$prefix/;
+                    my $candi = %$dist;
+                    $candi<files>{$file} = self!resources-dir.child($candi<files>{$file});
```

Comment:
Sorry, we cannot use self!resources-dir here, since that method would try to create a resources/ directory if it doesn't exist yet and we may only assume that $prefix is writeable in the install method. Violating this has broken system wide installations in 2015.12.